src/App.css

```diff
@@ -1,39 +1,3 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
```

Comment:
### Your Readme.md :arrow_down:
![readme](https://user-images.githubusercontent.com/93039359/188724935-4db54576-7a51-43fd-a2c5-659bfbf315c5.png)

- [ ] Please make your readme file descriptive because it is the documentation of your project which tells other developers or people about your project. You can use the [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) for this purpose :blush:

Reply:
Thank you very much! You were right! My README.md file was missing in the other branch because it had unresolved conflicts between branches and it didn't merge.